docs/source/components/nodes/stereo_depth.rst

```diff
@@ -64,6 +74,112 @@ as:
 the threshold get invalidated, i.e. their disparity value is set to zero. You can set the confidence threshold with :code:`stereo.initialConfig.setConfidenceThreshold()`.
 
+Calculate depth using dispairty map
+###################################
+
+Disparity and depth are inversely related. As disparity decreases, depth increases exponentially depending on baseline and focal length. Meaning, if the disparity value is close to zero, then small variations in depth generate great depth differences. Similarly, if disparity value is big, then small variations in depth do not generate depth differences.
+
+By considering this fact, depth can be calculated using this formula:
+
+.. code-block:: python
+
+  depth = focal_length_in_pixels * baseline / disparity_in_pixels
+
+where baseline is the distance between two mono cameras. Note the unit used for baseline and depth is the same.
+
+To get focal length in pixels, use this formula:
+
+.. code-block:: python
+
+  focal_length_in_pixels = image_width_in_pixels * 0.5 / tan(HFOV * 0.5 * PI/180)
+  
+  # With 400P mono camera resolution where HFOV=71.9 degrees
+  focal_length_in_pixels = 640 * 0.5 / tan(71.9 * 0.5 * PI / 180) = 441.25
+  
+  # With 800P mono camera resolution where HFOV=71.9 degrees
+  focal_length_in_pixels = 1280 * 0.5 / tan(71.9 * 0.5 * PI / 180) = 882.5
+
+Examples for calculating the depth value, using the OAK-D (7.5cm baseline):
+
+.. code-block:: python
+
+  # For OAK-D @ 400P mono cameras and disparity of eg. 50 pixels
+  depth = 441.25 * 7.5 / 50 = 66.19 # cm
+  
+  # For OAK-D @ 800P mono cameras and disparity of eg. 10 pixels
+  depth = 882.5 * 7.5 / 10 = 661.88 # cm
+
+Note the value of disparity depth data is stored in :code:`uint16`, where 0 is a special value, meaning that distance is unknown.
+
+Min stereo depth distance
+#########################
+
+If the depth results for close-in objects look weird, this is likely because they are below the minimum depth-perception distance of the device.
+
+To calcualte this miminum distance, use the :ref:`depth formula <Calculate depth using dispairty map>` and choose the maximum value for disparity_in_pixels parameter (keep in mind it is inveresly related, so maximum value will yield the smallest result).
+
+For example OAK-D has a baseline of **7.5cm**, focal_length_in_pixels of **882.5 pixels** and the default maximum value for disparity_in_pixels is **95**. By using the above formula we get:
+
+.. code-block:: python
+
+  min_distance = 882.5 * 7.5cm / 95 = 69.67cm
+
+or roughly 70cm.
+
+However this distance can be cut in 1/2 (to around 35cm for the OAK-D) with the following options:
+
+1. Changing the resolution to 640x400, instead of the standard 1280x800.
+
+2. Enabling Extended Disparity.
+
+This mode extends the disparity search to a maximum of 191 pixels from the standard 96 pixels, thereby 1/2-ing the minimum depth.
```

Comment:
can you be specific that in `In Exended mode it extends the disparity search to a maximum of 191 pixels from the standard 96 pixels` add also add why and how it expands that ?

Reply:
@saching13  IMO it's written clear enough so the `why` is answered - or we can write it like 
`This mode extends the disparity search to a maximum of 191 pixels from the standard 96 pixels, which achieves depth perception at 1/2 closer distances`. Not sure how to add the `how` (talking about internal workings)?

Reply:
In Extended mode we do get 191 levels of disparity but the thing is we don't do 191 pixels disparity search on the image. 
We do 96 on the  1280x720 resolution and 96 on the 640x360 resolution. and then merge them to make 191. and the reason why it is not 192 is because zero is common. 


Another thing in the above explaination is that we do two points and say both does 1/2-ing the min depth. So just to be clear that the next line refers to Extended mode we could just say it directly instead of indirect reference by using `This mode`